.github/workflows/publish.yml

```diff
@@ -74,8 +75,7 @@ jobs:
           latest: ${{ github.event.inputs.latest == 'true' && '--latest' || '' }}
         run: |
           python3 -m dcpy.connectors.edm.publishing publish \
-            -p db-${{ inputs.product }} \
-            -b ${{ inputs.draft_build }} \
-            -v ${{ inputs.publishing_version }} \
-            -a ${{ inputs.product_acl }} \
-            $latest
```

Comment:
You dropped this!

Reply:
Make sure you include the final backslash too, on the `--acl` line

Reply:
oops!